### ISSUE TITLE
IllusionDiffusion in local UI?

### ISSUE THREAD

**Issue by ke1ne** [rupeshs/diffusionmagic#10]
Hi, I'm almost sure I saw IllusionDiffusion implemented in your UI "locally". But it's gone. Did you remove it? Or it's my fake memory? Do you plan to add later?
Thanks for your work!

**Comment by rupeshs** [comment]
@ke1ne  It is currently in beta, it is available in the branch "add-illusion-diffusion" https://github.com/rupeshs/diffusionmagic/tree/add-illusion-diffusion. You can clone and checkout the branch.

**Comment by ke1ne** [comment]
@rupeshs Yeah, I see it, thanks. So it will go to the main branch later, right?

**Comment by rupeshs** [comment]
Yes, it will, one thing it does not support low VRAM.

**Comment by ke1ne** [comment]
@rupeshs What VRAM amount considered as low?

**Comment by rupeshs** [comment]
less than 9GB

**Comment by ke1ne** [comment]
Thanks!
